.github/workflows/ci.yml

```diff
@@ -51,3 +51,30 @@ jobs:
       - name: Generate FFI bindings
         run: just gen
       - run: cargo clippy --all-targets -- -D warnings
+
+  integration-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: extractions/setup-just@v1
+      - name: Setup rust toolchain
+        run: rustup show
+      - uses: Swatinem/rust-cache@v2.2.0
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: "stable"
+      - name: Install FFI bindings
+        run: cargo install flutter_rust_bridge_codegen
```

Comment:
🔧 We should fix the version of the `flutter_rust_bridge_codegen` here. Otherwise it will always install the latest, which will get out of sync with our flutter dependency which is currently locked to `1.62.0`

Reply:
Specifically we can use --version to set it